set.mm

```diff
@@ -214931,7 +214941,7 @@ is an important property of monoids (see ~ mndid ), and therefore also for
 
 $(
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-  Ordered sums in a magma
+  Group sum operation for magmas
```

Comment:
A clearer naming for that operation would be "Iterated composition" (to use also at other places).
And "*in* magmas" sounds better.

Reply:
I think this section header was changed already some time ago: In 2020, it was "Ordered group sum operation" (see #1457), but the term "ordered" was (and is still) not mentioned in the comment for df-gsum. The title "Ordered sums in a magma" was also discussed already in this Issue. The term "ordered", however, seems to be too restrictive, because gsum is also defined for arbitrary (unordered) finite sets, see ~gsumval3 (this may make sense only if all involved elements commute). Therefore, I would like to keep the term "group sum operation" for the moment, not only being applicable for groups (like the "group operation" `+g` is also not used for groups only, but also for magmas, monoids, etc. Therefore, I just would change the section header to "Group sum operation in magmas".

For a complete renaming, we should have a broader discussion (within a separate Issue which I will open soon). 

I think the proposed term "composition" comes from Bourbaki (Algebra I, definition 4 on page 3) Here, also the term "ordered" is used: "composition ... of the ordered sequence ...". Unlike `gsum`, however, this definition cannot be used for arbitrary finite sets. 

According to Wikipedia, "Iterated binary operation" would be an appropriate name (see https://en.wikipedia.org/wiki/Iterated_binary_operation), this would be consistent if  `+g` would be called "binary operation" (see also https://en.wikipedia.org/wiki/Binary_operation) instead of "group operation" (however, the term "binary operation" is already used in set.mm for arbitrary functions ( S X S ) -> M, not only for functions ( S X S ) -> S). In https://planetmath.org/binaryoperation, the term "internal composition" is used for  `+g`, whereas in https://www.mathyma.com/mathsNotes/index.php?lng=en&trg=S1C2_Struct_IntBinOp it is called "internal binary operation".

Reply:
Issue #3726 opened to continue this discussion.

Reply:
Thanks for opening the issue. So you'll keep that section title unchanged for now ? (I think "group sum operation for magmas" sounds bizarre.)